SpyFakku | Fixed SpyFakku

KenjieDec opened the pull request:
Closes #5294

Checklist:
- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

KenjieDec committed:
Fixed Spyfakku

AwkwardPeak7 commented on src/en/spyfakku/src/eu/kanade/tachiyomi/extension/en/spyfakku/SpyFakku.kt:
```diff
@@ -95,116 +97,152 @@ class SpyFakku : HttpSource() {
-        artist = artists?.joinToString { it.name }
-        genre = tags?.joinToString { it.name }
-        thumbnail_url = "$baseImageUrl/$hash/1/c"
+        url = "/g/$id?$pages&hash=$hash"
```
changing url means users would need to migrate, maybe fetch it in the page list methods(?) since it appears to be used there

KenjieDec replied:
Hello, sorry for the late reply. Does my recent change/commit align with what you're suggesting? Thank you.

KenjieDec committed:
Fixed circles, and others

- Fixed circles ( Original API's currently giving full circle list, not the specific comic circle )
- Added attempts for fetching manga details
- Apply AwkwardPeak's suggestion
